statsmodels/tsa/stattools.py

```diff
@@ -2134,6 +2136,141 @@ def _kpss_autolag(resids, nobs):
     autolags = int(gamma_hat * np.power(nobs, pwr))
     return autolags
 
+def rur(x, store=False):
```

Comment:
I think the name should change.  Possibilities are

```python
rur_unit_root_test
range_unit_root_test
```

There are probably some other sensible ones.  But `rur` is not discoverable -- it helps to have the test easy to find.